File: Array/1929-Concatenation-of-Array/1929-Concatenation-of-Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
    let ans = []
    let n = nums.length

    for(let i = 0;i < n*2; i++) {
        ans[i] = nums[i%n];
    }
    return ans;
};

File: Array/1929-Concatenation-of-Array/1929-Concatenation-of-Array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans(nums.size()*2);
        int n = nums.size();
        for(int i = 0;i<n*2;i++) {
            ans[i] = nums[i%n];
        }
        return ans;
    }
};
